src/components/imageOptionsEditor/imageOptionsEditor.js

```diff
@@ -90,9 +91,6 @@ import '../../elements/emby-input/emby-input';
     }
 
     async function showEditor(itemType, options, availableOptions) {
```

Comment:
```suggestion
    function showEditor(itemType, options, availableOptions) {
```

Reply:
Double check (I'm on mobile) But I can't see any awards any more

Reply:
it should probably return `Promise` because it is exported as `show`.

Reply:
Removed!